lib/engines/content_object_store/test/unit/app/models/content_block_schema_test.rb

```diff
@@ -27,11 +27,14 @@ class ContentObjectStore::SchemaTest < ActiveSupport::TestCase
       {
         "something" => {},
         "something_else" => {},
-        "content_block_foo" => {
+        "content_block_email_address" => {
```

Comment:
I think I added foo here because I didn't want to tie these tests to the interface of email address. That said, it is more realistic and seems stable enough so perhaps all is well 👍

Reply:
In fact, does the fact this change is required point to the fact that our test suite may be dependent on loading real schemas from Publishing API? If so, I think we ought to stub that out instead and these tests might be being helpful?

Reply:
The reason I changed this was because the block types now need to be valid schemas (as per `ContentObjectStore::ContentBlockSchema::VALID_SCHEMAS`). That said, I'd feel happier if there was a way we could make it feel a bit more fake in the tests. I'll have a think 🤔

Reply:
Sorted! Have broken the logic for defining what a valid schema is into a method on its own, so we can stub it out when testing 👍